tests/graph_rust.rs

```diff
@@ -0,0 +1,53 @@
+use graph_rust::{Bfs, Dfs, Graph, GraphType};
+
+#[test]
```

Comment:
Unit test can be just in the file that defines the structure to be tested using 
```
mod test {

}
```

Put this to `lib.rs`. Please read Rust tests carefully to see what should be properly tested.